frontend/package.json

```diff
@@ -6,26 +6,26 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "test:unit": "vue-cli-service test:unit",
-    "lint": "vue-cli-service lint && npm run license-file-headers-add",
+    "lint": "vue-cli-service lint",
     "prettify": "prettier --write **/*.{css,html,js,json,mjs,scss,ts,tsx,yml,md,vue}",
     "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(js|vue)' --locales './src/locales/**/*.json'",
     "license-file-headers-add": "license-check-and-add add -f license-header/config.json -r 2021",
-    "license": "generate-attribution -o ./licenses",
+    "license": "generate-attribution -o ./licenses && npm run prettify",
```

Comment:
why not put this under prettify?

Reply:
Good idea, will apply this in a new commit